packages/mask/src/extension/popups/pages/Wallet/ImportWallet/index.tsx

```diff
@@ -269,16 +269,15 @@ const ImportWallet = memo(() => {
                     </Tabs>
                     <TabPanel value={tabs.mnemonic} className={classes.tabPanel}>
                         <StyledInput
-                            multiline
+                            type="password"
```

Comment:
Why remove `multiline` ?

Reply:
password input does not support multiline